src/App.tsx

```diff
@@ -33,25 +40,62 @@ export const PATHS = {
   WELCOME_PAGE: "/welcome",
 };
 
+const queryClient = new QueryClient({
```

Comment:
I see the documentation for this constructor at: https://tanstack.com/query/latest/docs/reference/QueryClient

The [network mode docs](https://tanstack.com/query/latest/docs/framework/react/guides/network-mode#network-mode-online) say the default network mode is `online`.

> Since TanStack Query is most often used for data fetching in combination with data fetching libraries, the default network mode is [online](https://tanstack.com/query/latest/docs/framework/react/guides/network-mode#network-mode-online).